src/context/WalletProvider/NewWalletViews/NewWalletViewsSwitch.tsx

```diff
@@ -337,7 +337,7 @@ export const NewWalletViewsSwitch = () => {
         onClose={onClose}
         isCentered
         trapFocus={false}
-        closeOnOverlayClick={false}
+        closeOnOverlayClick={true}
```

Comment:
Not sure why we weren't doing this but that's infuriating to not being able to click away and close, now we can! 🎉

Reply:
I think if it's in case an user click away by mistake while he was currently creating a wallet which could be really boring for them